static/styles/recent_topics.scss

```diff
@@ -69,5 +69,47 @@
             // is set to header's width.
             width: 1px;
         }
+
+        .recent_avatars {
+            display: inline-flex; /* Causes LI items to display in row. */
+            list-style-type: none;
+            margin: auto; /* Centers vertically / horizontally in flex container. */
+            height: 15px !important;
+
+            /*
+                By using the row-reverse layout, the visual ordering will be opposite of
+                the DOM ordering. This will allows us to stack the items in the opposite
+                direction of the natural stacking order without having to mess with the
+                zIndex value. The MAJOR DOWNSIDE is that the HTML itself now reads
+                backwards, which super janky.
+            */
+            flex-direction: row-reverse;
+        }
+
+        .recent_avatars__item {
+            height: 15px;
+            margin: 0px 0px 0px 0px;
+            padding: 0px 0px 0px 0px;
+            position: relative;
+            width: 15px; /* Forces flex items to be smaller than their contents. */
+        }
```

Comment:
These feel too small.  Maybe 24px?  

Do we need to repeat the sizing here in addition to in the next block?

Reply:
I think @aero31aero did a better version of this:
![image](https://user-images.githubusercontent.com/25124304/82914302-f867a000-9f8c-11ea-9a93-48ec79de992e.png)
@aero31aero  Can you send the link to your commit for this, I will cherry-pick it here at the end. (or we can do it once an initial UI for this gets merged?

Reply:
We should never display avatars as circles; it looks bad with some images (E.g. bot avatars which are usually product logos).  But otherwise that lgtm.